python/ray/scripts/scripts.py

```diff
@@ -747,8 +750,18 @@ def start(
         cli_logger.newline()
         with cli_logger.group("Next steps"):
             dashboard_url = node.address_info["webui_url"]
-            if bootstrap_address.startswith("127.0.0.1:"):
-                if ray_constants.ENABLE_RAY_CLUSTER:
+            if ray_constants.ENABLE_RAY_CLUSTER:
+                cli_logger.print("To add another node to this Ray cluster, run")
+                # NOTE(kfstorm): Java driver rely on this line to get the address
+                # of the cluster. Please be careful when updating this line.
+                cli_logger.print(
+                    cf.bold(" {} ray start --address='{}'"),
+                    f"{ray_constants.ENABLE_RAY_CLUSTERS_ENV_VAR}=1"
```

Comment:
is this env var still useful?

Reply:
yea, if you want to start a multi-node mac cluster, you need this env var.

Reply:
So, I'd like to understand the user journey here. Is this correct?

1. win/mac users start ray using ray start --head
2. they connect a worker node using ray start --address
3. Get an error message that it needs an env var when starting ray start --head, and it is not recommended and not tested.
4. users start env_var ray start --head

Reply:
Yes. Also with this PR, the output of `ray start --head` won't contain instructions about how to create a worker node if it's osx or windows (it's hidden since it's not supported).